.circleci/workflows.yml

```diff
@@ -1786,7 +1821,45 @@ workflows:
           requires:
             - release_dist <<# pipeline.parameters.dry_run >> - Dry Run<</ pipeline.parameters.dry_run >>
 
-  # -- typically this workflow is executed on pull requests events for Community Edition Gravitee Repositories
+  release-hotfix-version:
+    when:
+      and:
+        - equal: [ release-hotfix-version, << pipeline.parameters.gio_action >> ]
+    jobs:
+      - setup:
+          name: setup
+          context: cicd-orchestrator
+      - release:
+          name: release_dist <<# pipeline.parameters.dry_run >> - Dry Run<</ pipeline.parameters.dry_run >>
+          context: cicd-orchestrator
+          dry_run: << pipeline.parameters.dry_run >>
+          graviteeio_version: << pipeline.parameters.graviteeio_version >>
+          hotfix_version: << pipeline.parameters.hotfix_version >>
```

Comment:
**suggestion**: as we are calling the action 'release-hotfix-version' maybe we can force the param to true here. wdyt?